lib/beaker-pe/install/pe_utils.rb

```diff
@@ -187,7 +195,11 @@ def frictionless_agent_installer_cmd(host, opts, pe_version)
 
             cmd = %Q{powershell -c "cd #{host['working_dir']};#{cert_validator};\\$webClient = New-Object System.Net.WebClient;\\$webClient.DownloadFile('https://#{downloadhost}:8140/packages/current/install.ps1', '#{host['working_dir']}/install.ps1');#{host['working_dir']}/install.ps1 -verbose #{frictionless_install_opts.join(' ')}"}
           else
-            curl_opts = %w{--tlsv1 -O}
+            if require_tlsv1?(host)
+              curl_opts = %w{--tlsv1 -O}
```

Comment:
if you deal with the `-0` option separately, then you can add the tls test & option in a much more readable way. If I'm recalling how `%w` works properly, then it would look like this:
```ruby
curl_opts = ["-0"] # or %w{-0} as below
curl_opts << "--tlsv1" if require_tlsv1?(host)
```

Reply:
updated